src/components/Card/CardContainer.jsx

```diff
@@ -1,5 +1,20 @@
+import Card from "./Card";
+import articles from "../../utils/URLs.json";
+
 function CardContainer() {
-  return <main></main>;
+  return (
+    <main className="grid min-w-[40rem] justify-center grid-cols-10 gap-6 bg-fixed bg-right-bottom bg-no-repeat justify-items-center bg-default mx-80">
+      {articles.map((article) => (
+        <Card
+          key={article.createDate}
+          favicon={article.faviconUrl}
+          domain={article.siteName}
+          articleTitle={article.title}
+          readingTime={article.readingTime}
+        ></Card>
```

Comment:
내용이 없는 경우 self-closing 하는 것이 컨벤션에 더 적합해 보입니다!
```suggestion
        <Card
          key={article.createDate}
          favicon={article.faviconUrl}
          domain={article.siteName}
          articleTitle={article.title}
          readingTime={article.readingTime}
        />
```

https://github.com/airbnb/javascript/tree/master/react#tags